app/components/markdown/markdown.tsx

```diff
@@ -38,7 +38,6 @@ import type {
 type MarkdownProps = {
     autolinkedUrlSchemes?: string[];
     baseTextStyle: StyleProp<TextStyle>;
-    baseParagraphStyle?: StyleProp<TextStyle>;
```

Comment:
Not used?

Reply:
Only used on permalink errors and was not really needed, apparently.